index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title> PRELOG </title>
+    <script src="https://use.fontawesome.com/23bea62652.js"></script>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    
+  </head>
+  <body>
+    
+    <!-- start the Header section of the website-->
+    <nav>
+    <header >
+        <img class="logo" src="./images/logo (1).png" width="190px" height="43.7" alt="logo" />
```

Comment:
remove the inline style from here (width and height). All the styles should be in the `style.cc` file